src/App.css

```diff
@@ -28,6 +28,49 @@
   color: #61dafb;
 }
 
+.my-task {
+  padding-left: 50px;
+  width: 250 px;
+  height: 50px;
+  border: solid;
+  display: flex;
+  justify-content: center;
+}
+
+.row {
+  display: table;
+  width: 800px;
+  padding: 10px;
+}
+
+.col1 {
+  float: left;
+  width: 50px;
+}
+
+.box1 {
+  border: 1px solid;
+  width: 40px;
+  border-radius: 25px;
+  display: inline-block;
+  text-align: center
+}
```

Comment:
Can you double-check the `width`? I think it's `48px` in the design
Also, the `border-radius` is only `10px` in the design too.

Reply:
And I don't think we need that `display: inline-block`